pkg/securelaunch/policy/vm_test.go

```diff
@@ -0,0 +1,223 @@
+	"github.com/hugelgupf/vmtest/guest"
+	"github.com/hugelgupf/vmtest/qemu"
+	slaunch "github.com/u-root/u-root/pkg/securelaunch"
+)
+
+const policyStr = `{
+    "collectors": [
+      {
+        "type": "dmi",
+        "events": [
+          {
+            "label": "BIOS",
+            "fields": []
+          },
+          {
+            "label": "System",
+            "fields": []
+          },
+          {
+            "label": "Processor",
+            "fields": []
+          }
+        ]
+      },
+      {
+         "type": "files",
+         "paths": [ "sda1:/opc/foo" ]
+      },
+      {
+         "type": "storage",
+         "paths": [ "sda1" ]
+      },
+      {
+         "type": "cpuid",
+         "location": "sda2:/cpuid.txt"
+      }
+    ],
+    "attestor": {},
+    "launcher": {
+        "type": "kexec",
+        "boot entries": {
+            "0": {
+                "kernel name":"vmlinuz-5.4.17-2036.103.3.el7uek.x86_64",
+                "kernel hash":"59c762615cdb09558bcd80d3025d023b436386fd9ab6e09d709418fbbce7680c",
+                "initrd name":"initramfs-5.4.17-2036.103.3.el7uek.x86_64.img",
+                "initrd hash":"a39a6ba3e35dffd0b91ca0f0dee2a7bfb16a447353746cf83d6dc7139dc99c4a",
+                "cmdline":"BOOT_IMAGE=/vmlinuz-5.4.17-2036.103.3.el7uek.x86_64 root=/dev/mapper/ol_ol7--sl-root ro crashkernel=auto rd.luks.uuid=luks-06f28824-6b55-4219-b1a4-69a466af670b rd.lvm.lv=ol_ol7-sl/root rd.lvm.lv=ol_ol7-sl/swap console=ttyS0,115200"
+            },
+            "1": {
+                "kernel name":"vmlinuz-4.14.35-1902.303.5.3.el7uek.x86_64",
+                "kernel hash":"fa17071a44c0c185de9f879cddf6823f4d64a0c26604657655dad7c1d2fae39c",
+                "initrd name":"initramfs-4.14.35-1902.303.5.3.el7uek.x86_64.img",
+                "initrd hash":"c409a5118dacb1c2c71b9dab078ff670f15cae5219475fba902b508dea616187",
+                "cmdline":"console=ttyS0,115200n8 earlyprintk=serial,ttyS0,115200n8,keep printk.time=y"
+            }
+        }
+    },
+    "eventlog": {
+        "type": "file",
+        "location": "/evtlog"
+    }
+}
+`
+
+const pubkeyStr = `
+-----BEGIN PUBLIC KEY-----
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAmsvG6Bj+cylwEmrUrqKj
+W7lvovDDjJxizSrFTizd9gid/V9AT8mbwwoJJp2S2WVdGHPn3hDuFlyQvKTkqQlE
+6l1+ALEiAxjiGrKz6c+x21X1thHv01w9/NX/f7K3B1QNnj972k96z6PW0jjafxYe
+oc+ylGEqh4GxOlYbcdELdXi261+n+I0CsDJEzLVJnbs7YGW0dPN4UlGIAoVe4O4x
+Taz74TBJEODD6GrNLpZeDD2ke0KwZSjMF2lDBX40Oyj/yn10dYUqKvfODNvWYIQg
+fRCANXtNC3u89wU0HamJj8agITLOqIxp2UOaBj0qF4lAN4PP+pF6+6r0bngmfy0w
+3hnEFVplySGp/S94071hR7zAMd4ZSIACcUQjFvt5BqmHtXgDRviJc4lh8niZ7cg0
+ID8PgLjn+wLL5EZKsztWsk4030TeGtZ82eMiczqpsj7/5SWsxRBUraEi5aD6CREG
+LIRbeVynOqC8KPXWLWDUvZSgwWUOYykZHYY9qCU4UgWNAx/uOfl4ZHEVK4LtrNI0
++byI7V27o9f5uhz5XJiAjObTAyh31xP2p4Xu6wrCQy1E4i1l9gTsCl6rQAPY0K+A
+U8CAmAOKf2E5jEe8itnnsKu6Ii46ndLwJrBGgjsEYRNVln04owEiTsPrUCTPBS3O
+RaNetBXso+Hgdg8M25WgCI8CAwEAAQ==
+-----END PUBLIC KEY-----
+`
+
+const signatureStr = "" +
```

Comment:
Maybe it better to embed these files?

Reply:
It might be, but this PR is a "catch up" PR and I would prefer to do that sort of thing once it is working. It is long enough already.